db.js

```diff
@@ -256,6 +259,15 @@ export class ActivityPubDB extends EventTarget {
     const actor = await this.getActor(url)
     console.log('Actor received:', actor)
 
+    // Add 'following' to timeline if the actor is followed
+    const isFollowing = await this.isActorFollowed(url)
+    if (isFollowing) {
+      for await (const note of this.searchNotes({ attributedTo: actor.id })) {
+        note.timeline.push('following')
```

Comment:
Only push if the note doesn't have 'following' in timeline already.

Let's add some consts at the top for `TIMELINE_ALL = 'all'` and `TIMELINE_FOLLOWING = 'following'` to reduce accidental misspelling